### PR TITLE
rpc: export unexported fields of public structures

### LINKED ISSUE #349: Ensure all fields of returned exported structures are public
All structures returned by contracts should have only public fields. It will save us from unaccessible RPC binding structure fields like
https://github.com/nspcc-dev/neofs-contract/blob/84ff5d244f6913987c2fb7dc3e2adfb5dfa81381/rpc/container/rpcbinding.go#L27-L32
Until the https://github.com/nspcc-dev/neo-go/issues/3083 is resolved. And in general, it's just a good practice.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
And upgrade NeoGo to fetch the generator bugfix.

Close #349.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ export GOBIN ?= $(shell pwd)/bin
 export CGO_ENABLED=0
 NEOGO ?= $(GOBIN)/cli
 VERSION ?= $(shell git describe --tags --dirty --match "v*" --always --abbrev=8 2>/dev/null || cat VERSION 2>/dev/null || echo "develop")
-NEOGOORIGMOD = github.com/nspcc-dev/neo-go@v0.101.5-0.20230802075307-afa4530c7d73
+NEOGOORIGMOD = github.com/nspcc-dev/neo-go@v0.101.5-0.20230810175032-d425df855e32
```
Comment (roman-khimov) [inline Makefile:10]: I'd wait for it to settle in the `master` (shouldn't take a lot of time).

**Review by roman-khimov — COMMENTED** [review]
OK otherwise.

File: audit/audit_contract.go [hunk 2]
```diff
@@ -12,10 +12,10 @@ import (
 )
 
 type (
-	auditHeader struct {
-		epoch int
-		cid   []byte
-		from  interop.PublicKey
+	AuditHeader struct {
+		Epoch int
+		Cid   []byte
```
Comment (roman-khimov) [inline audit/audit_contract.go:17]: CID then.

File: container/container_contract.go [hunk 3]
```diff
@@ -39,9 +39,9 @@ type (
 		Size int
 	}
 
-	containerSizes struct {
-		cid         []byte
-		estimations []Estimation
+	ContainerSizes struct {
+		Cid         []byte
```
Comment (roman-khimov) [inline container/container_contract.go:43]: Ditto

File: Makefile (hunk 1 above)
Comment (AnnaShaleva) [reply Makefile]: Fixed.

File: audit/audit_contract.go (hunk 2 above)
Comment (AnnaShaleva) [reply audit/audit_contract.go]: Fixed.

File: container/container_contract.go (hunk 3 above)
Comment (AnnaShaleva) [reply container/container_contract.go]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**